pkg/pods/pod.go

```diff
@@ -378,7 +378,7 @@ func (pod *Pod) Verify(manifest *Manifest, keyring openpgp.KeyRing) error {
 
 		var launchableDigest map[string]string
 
-		if stanza.DigestSignatureLocation == "" || keyring == nil {
+		if stanza.DigestSignatureLocation == "" || verifier == nil {
```

Comment:
we want people to be able to deploy the preparer without needing to gpg sign things, so if there is not a `keyring` key in the preparer's config then it shouldn't do signature checks. Right now we do that by checking if the verifier is nil everywhere, but another idea would be to have a stand in verifier that is always happy